probert/tests/helpers.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import random
+import string
+import testtools
+
+class CiTestCase(testtools.TestCase):
+
+    @classmethod
+    def random_string(cls, length=8):
```

Comment:
We haven't had this conversation in the context of probert, but ***if*** we want to migrate to pytest in the future, we'll find it easier if helpers (that don't require `self`) aren't attached to test classes.

Could this be a function that `test_multipath.py` just imports directly?

Reply:
It can, I'll move it.